crocodealer.cabal

```diff
@@ -25,11 +25,16 @@ library
   exposed-modules:     Crocodealer
                          Crocodealer.Config
                          Crocodealer.Core.Label
+                         Crocodealer.Command.Sync
+                         Crocodealer.Command.Colors
                        Prelude
 
   build-depends:       base-noprelude ^>= 4.12.0.0
                      , relude ^>= 0.5.0
                      , tomland ^>= 1.1
+                     , http-conduit ^>= 2.3.7.3
```

Comment:
`http-conduit` is a bit heavyweight library. We're usually using `http-client` + `http-client-tls` to download files. Usage code can be found here:

* https://github.com/kowainik/issue-wanted/blob/2b1c287235998b3dbb5de92aaeef25a28b3d83b5/src/IW/Effects/Download.hs#L33-L48

Though, it uses a slightly different interface. Though this refactoring to more lightweight libraries can be done separately, I'm okay with this.

Reply:
Sure, I was looking for a more lightweight library, will address this in a separate commit.